commands/import.go

```diff
@@ -314,3 +330,114 @@ func jobListCmdF(c client.Client, command *cobra.Command, jobType string) error
 func importJobListCmdF(c client.Client, command *cobra.Command, args []string) error {
 	return jobListCmdF(c, command, model.JobTypeImportProcess)
 }
+
+type Statistics struct {
+	Schemes        int `json:"schemes"`
+	Teams          int `json:"teams"`
+	Channels       int `json:"channels"`
+	Users          int `json:"users"`
+	Emojis         int `json:"emojis"`
+	Posts          int `json:"posts"`
+	DirectChannels int `json:"direct_channels"`
+	DirectPosts    int `json:"direct_posts"`
+	Attachments    int `json:"attachments"`
+}
+
+func importValidateCmdF(command *cobra.Command, args []string) error {
+	configurePrinter()
+
+	defer printer.Print("Validation complete\n")
+
+	injectedTeams, err := command.Flags().GetStringArray("team")
+	if err != nil {
+		return err
+	}
```

Comment:
Just a small arrangement:
```suggestion
	defer printer.Print("Validation complete\n")

	injectedTeams, err := command.Flags().GetStringArray("team")
	if err != nil {
		return err
	}

```